### DISCUSSION TITLE
clear url

### DISCUSSION THREAD

**Discussion by Cootmaster** [smplayer-dev/smplayer#355] [Q&A]
is there  a way
![image1](https://user-images.githubusercontent.com/80638263/150689781-24479176-abc1-4879-b0aa-70627b21f168.jpg)
 to clear the url    globe  CONTL U  
all i see is  a x  and pull down   arrow
to show  multiple urls
www xxx'\
wwww xxx
i dont see a way to completly   clear rhe line  and delete everything in the pull down menu

**Comment by smplayer-dev** [comment]
Preferences -> Interface -> Privacy, set Max. Items for URLs to 0.

**Comment by Cootmaster** [comment]
great   now shows the one im viewing ty
